gensrc/thrift/AgentService.thrift

```diff
@@ -274,7 +274,8 @@ struct TRecoverTabletReq {
 enum TTabletMetaType {
     PARTITIONID,
     INMEMORY,
-    ENABLE_PERSISTENT_INDEX
+    ENABLE_PERSISTENT_INDEX,
+    WRITE_QUORUM
```

Comment:
The option is for table, not for one load job?

Reply:
yes